scripts/local/native/native.sh

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env bash
```

Comment:
I think the creation of the "native" directory here is likely to cause some unnecessary confusion. I think this script is a local integration test right?